Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8.9
+#note docker node image comes with yarn
+
+WORKDIR /agriculture-core/
+
+#ADD . /agriculture-core/
+
+RUN yarn install --frozen-lockfile
+
+RUN npm run build
```

Comment:
This gets locked to the image every time you build, so you would have to rebuild the image every time you make changes. So this should be taken out, and just be a different way to run this file. 

Example
https://github.com/enochtsang/personal-website/blob/master/docker-build.sh

Here in the bash file, there are two ways to run the image, running or building, we'd want to do something similar.